graylog2-server/src/main/resources/org/graylog2/security/csp.config

```
@@ -0,0 +1,34 @@
+######################################################
+# GRAYLOG SECURITY CONTENT POLICY CONFIGURATION FILE #
+######################################################
+#
+# Use this file to configure additional CSP directive values. These will be merged with
+# values specified via the @CSP annotation, if the group name matches.
+#
+# Entries are generally expected to be a single line of the form, one of the following:
+#
+# groupName.directive=value
+#
+# White space that appears between the directive and value is ignored,
+# so the following are equivalent:
+#
+# test.name=Stephen
+# test.name = Stephen
+#
+# White space at the beginning of the line is also ignored.
+# Lines that start with the comment characters ! or # are ignored. Blank lines are also ignored.
+# The value is generally terminated by the end of the line. White space following the
+# value is not ignored, and is treated as part of the value.
+#
+# Example:
+# default.img-src=my_imgsrc.com
+# default.connect-src=my_connectsrc.com:9999
+
+default.default-src='self'
+default.style-src='self' 'unsafe-inline'
+default.script-src='strict-dynamic' 'nonce-{nonce}' 'unsafe-eval'
+default.img-src=data:*
```

This should be `data:` instead and we should also include `'self'` and the exception for OSM here:
```suggestion
default.img-src=data: 'self' https://*.tile.openstreetmap.org
```

---

updated